hasher-matcher-actioner/webapp/src/Api.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+// TODO figure out root path. For now copy your own URL gateway here
+const API_ROOT = 'https://<APPID>.execute-api.us-east-1.amazonaws.com'
+// TODO replace with real auth someday
+const API_TOKEN = '<TOKEN>'
```

Comment:
For now, you can hardcode it with the placeholder, unless this is the new placeholder